java-checks/src/main/java/org/sonar/java/checks/SwitchCaseWithoutBreakCheck.java

```diff
@@ -76,7 +76,7 @@ public void visitCaseGroup(CaseGroupTree tree) {
     super.visitCaseGroup(tree);
 
     if (invalidCaseGroups.remove(tree)) {
-      context.addIssue(Iterables.getLast(tree.labels()), this, "End this switch case with an unconditional break, return or throw statement.");
+      context.reportIssue(this, Iterables.getLast(tree.labels()), "End this switch case with an unconditional break, return or throw statement.");
```

Comment:
@benzonico I would raise the issue on `Iterables.getLast(tree.labels()).caseOrDefaultKeyword()`.